### PR TITLE
Check for errors from Trillian.

### PR DESCRIPTION
**Pull request by benlaurie**

### CODE REVIEW

**Comment by Martin2112** [comment]
Sorry can you rebase and push again? I was playing with setting up travis and I think it's got confused.

**Comment by benlaurie** [comment]
Sure, I have to update it slightly anyway.

File: etherslurp/follower/follower.go [hunk 1]
```diff
@@ -116,10 +117,15 @@ nextAttempt:
...
-			if _, err := f.tc.QueueLeaves(ctx, &trillian.QueueLeavesRequest{LogId: f.logID, Leaves: []*trillian.LogLeaf{leaf}}); err != nil {
+			r, err := f.tc.QueueLeaf(ctx, &trillian.QueueLeafRequest{LogId: f.logID, Leaf: leaf})
+			if err != nil {
 				glog.Errorf("Failed to Queue block %v: %v", nextBlock, err)
 				continue nextAttempt
 			}
+			c := codes.Code(r.QueuedLeaf.GetStatus().GetCode())
+			if c != codes.OK {
+				glog.Errorf("Leaf add failed: %s", r.QueuedLeaf.GetStatus())
+			}
```
Comment (RJPercival) [inline etherslurp/follower/follower.go]: Do you still want the "Copied to %v" message below to print even when leaf add failed?

**Comment by mhutchinson** [comment]
This isn't needed when #257 is merged. Closing this, with my head bowed in respect for how long this PR has been pending.

**Pull request closed without merging**